oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/util.js

```diff
@@ -6,6 +6,7 @@ import { getInputFromDOM } from "@js/oarepo_ui";
 import { FormConfigProvider } from "./contexts";
 import { Container } from "semantic-ui-react";
 import { BrowserRouter as Router } from "react-router-dom";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
```

Comment:
We are using new dependency here, that is not in `webpack.py`, please add it

Reply:
Sorry, I had it in oarepo-vocabularies instead, I put it in oarepo now:)